packages/components/src/utils/validation/partials/validatePartialUsage.spec.ts

```diff
@@ -280,49 +284,49 @@ describe('validateGetLoaderScriptUsage()', () => {
 });
 
 describe('validateGetInitialStylesUsage()', () => {
-  it('should call document.querySelector() with correct parameters', () => {
-    jest.spyOn(helperUtils, 'getPorscheDesignSystemPrefixesForVersions').mockReturnValue({
-      '1.2.3': [''],
-      '1.2.4': ['prefix'],
-      '1.2.5': ['my-prefix'],
-    });
+  it('should call document.head.querySelector() with correct parameters', () => {
     const spy = jest.spyOn(document.head, 'querySelector');
+    jest.spyOn(validatePartialUsageUtils, 'throwPartialValidationError').mockImplementation(); // mocked since it throws an exception
 
     validateGetInitialStylesUsage();
 
     expect(spy).toBeCalledWith('style[data-pds-initial-styles]');
     expect(spy).toBeCalledTimes(1);
   });
 
-  it('should call logPartialValidationWarning() with correct parameters when initial style is not found', () => {
-    jest.spyOn(helperUtils, 'getPorscheDesignSystemPrefixesForVersions').mockReturnValue({
-      '1.2.3': [''],
-      '1.2.4': ['prefix'],
-      '1.2.5': ['my-prefix'],
-    });
-    const spy = jest.spyOn(validatePartialUsageUtils, 'logPartialValidationWarning');
+  it('should call throwPartialValidationError() with correct parameters when initial style is not found', () => {
+    const spy = jest.spyOn(validatePartialUsageUtils, 'throwPartialValidationError').mockImplementation(); // mocked since it throws an exception
```

Comment:
Why not use expect toThrow()? In the other tests as well ([https://jestjs.io/docs/expect#tothrowerror](https://jestjs.io/docs/expect#tothrowerror))

Reply:
Hmm, that's covered by the `throwException()` tests. Here we cover that the correct function is called.
Testing for additional behavior would be redundant I guess and not the responsibility of the function under test?